server.go

```diff
@@ -286,6 +286,9 @@ func (s *server) channelHandleWrapper(handler ChannelHandler, handlerFunc Channe
 
 		// read the bytes from our body so we can create a channel log for this request
 		response := &bytes.Buffer{}
+
+		// Trim out cookie header, should never be part of authentication and can leak auth to channel logs
+		r.Header.Del("Cookie")
```

Comment:
any argument for redacting this instead of removing it?

Reply:
I thought about it but that was 3 lines of code instead of one so figured why bother. ;)